feature/oauth/src/main/java/com/teamsolply/solply/oauth/GoogleLoginHelper.kt

```
@@ -0,0 +1,88 @@
+        when (val credential = result.credential) {
+            is CustomCredential -> {
+                if (credential.type == GoogleIdTokenCredential.TYPE_GOOGLE_ID_TOKEN_CREDENTIAL) {
+                    try {
+                        val googleIdTokenCredential =
+                            GoogleIdTokenCredential.createFrom(credential.data)
+                        val idToken = googleIdTokenCredential.idToken
+//                        Log.d(TAG, idToken) //토큰
+//                        Log.d(TAG, googleIdTokenCredential.id) //이메일
+//                        googleIdTokenCredential.displayName?.let { Log.d(TAG, it) } //이름
+                        onSuccess(idToken) // 성공 시 처리 함수, 서버 응답 후 실행, 여기서는 테스트를 위해 이곳에서 실행
+                    } catch (e: GoogleIdTokenParsingException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)